coreapi/v23.0/beacon/api/vrf.go

```diff
@@ -1,8 +1,9 @@
 package api
 
 import (
-	"github.com/oasisprotocol/nexus/coreapi/v23.0/consensus/api/transaction"
 	"github.com/oasisprotocol/oasis-core/go/common/crypto/signature"
+
+	"github.com/oasisprotocol/nexus/coreapi/v23.0/consensus/api/transaction"
```

Comment:
aw no more separate goimports commit

Reply:
I tried hard to not have it! Buncha rebasing. Still ended up with some weird conflicts and I think one small post-fix commit :/